src/components/search/SearchResults.jsx

```diff
@@ -1,8 +1,99 @@
-const SearchResults = () => {
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import AgeIcon from '../../assets/icons/icon-age.svg';
+import LessonTypeIcon from '../../assets/icons/icon-lesson.svg';
+import ScheduleIcon from '../../assets/icons/icons-schedule.svg';
+
+const SearchResults = ({ classes, currentPage, totalPages, onPageChange }) => {
+  const navigate = useNavigate();
+  const handleClick = () => {
+    navigate('/register');
```

Comment:
I think we need navigate user to login page.

Reply:
Thank you Valya! Got it